main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/ministryofjustice/opg-search-service/internal/cache"
 	"github.com/ministryofjustice/opg-search-service/internal/cmd"
+	deletePackage "github.com/ministryofjustice/opg-search-service/internal/delete"
```

Comment:
ha, maybe we should call the package `remove` to avoid being a keyword?

Reply:
I originally called it `deindex` to avoid this, but I figured it was unnecessary obsfuscation just to avoid a keyword